### DISCUSSION TITLE
Janapriyam TV

### DISCUSSION THREAD

**Discussion by Manipulativemachan** [iptv-org/community#341] [Show and tell]
**_Channel Name:_** Janapriyam
**_Channel ID:_** JanapriyamTV.in
**_Country:_** IN
**_Broken Link (from playlist):_** https://jio.instream.ml/jio.php?c=Janapriyam_News&e=.m3u8&q=400
**_Possible Replacement (optional):_** xxx
**_Notes (optional):_** xxx

**Comment by BellezaEmporium** [comment]
From their website : https://cdn.janapriyam.in/janapriyam_jio/index.m3u8 (Not found)
